### ISSUE TITLE
screen readers cannot access the datalist polyfil using IE11

### ISSUE THREAD

**Issue by SBleeks** [wet-boew/wet-boew#7987]
When using either JAWS (latest version) and/or NVDA screen readers, the autocomplete list is not accessible. In other words the screen readers cannot read the automatically populated list of results even when using forms mode on/off with JAWS or NVDA's equivlant "browse mode".

This issue exists only with IE 11. There is no issue accessing the list using Chrome and Firefox with both JAWS and NVDA.

**Comment by nschonni** [comment]
IE11 has native (but buggy) support for the `datalist` element, so the WET plug-in shouldn't be loading at all https://caniuse.com/#feat=datalist

**Comment by LaurentGoderre** [comment]
Hmmm, I didn't know that

**Comment by CoolColin** [comment]
We should consider swapping out this widget from WET altogether and replace with this solution here that works extremely well in all browsers.. Thoughts?
https://haltersweb.github.io/Accessibility/autocomplete.html

**Comment by nschonni** [comment]
Yeah, that might be good as a new compenent/polyfill. We call it an autocomplete in the examples, but it isn't quite the same behaviour/element that we polyfill in that example.
Little bit of a related discussion https://github.com/w3c/wcag21/issues/486
